styles/styles.css

```diff
@@ -97,13 +103,13 @@
   padding: 0;
   margin: 0;
   position: relative;
-/* <<<<<<< 40-update-carousel-arrowshomepage
+  /* <<<<<<< 40-update-carousel-arrowshomepage
   background-color: var(--bg-color);
   background-image: url('../assets/hero_boy-semi-final-2.png'); */
-/* ======= */
-  background-color: rgba(46, 45, 51, .6);
+  /* ======= */
+  background-color: rgba(46, 45, 51, 0.6);
   background-image: url("../assets/hero_boy-semi-final-2.png");
-/* >>>>>>> main */
+  /* >>>>>>> main */

```

Comment:
I think this  may need to be addressed, it looks like it was a merge conflict that wasn't dealt with quite right but its not a big deal at this moment